### DISCUSSION TITLE
Stand alone Rasp-pi + 4mics

### DISCUSSION THREAD

**Discussion by gutocarv** [hcfman/sbts-aru#7] [General]
Hi, congrats on this amazing project
Do you think it can run in a just a stand alone centralized rasp-pi unit - and 4 mics arranged as a planar square - kind of 15meters - using four identical mic cable (same size, same specs  )? 
had aver tested this kind of configuration??

**Comment by hcfman** [comment]
Thanks for the compliment. The very short answer is I haven’t tried it but the code will certainly support recording multiple streams, it’s just different parameters and maybe some difference in the startup of jackd I would think.

However, technically speaking USB cable has a max length which I think will make 15m not work.

furthermore though. If you really needed the 15m square you could in principle use multiple Pi’s as the clocks are synched to sub microsecond accuracy.

Sounds like you are up to something interesting ?
